src/main/java/xyz/nucleoid/plasmid/entity/CustomEntity.java

```diff
@@ -13,6 +13,7 @@
 
 import javax.annotation.Nullable;
 
+@Deprecated
```

Comment:
Some deprecation comments would be nice here, so devs know what to use instead :D